ci-automation/base_sysexts.sh

```diff
@@ -8,4 +8,5 @@ fi
 ciabs_base_sysexts=(
     'containerd-flatcar:app-containers/containerd'
     'docker-flatcar:app-containers/docker&app-containers/docker-cli'
+    'incus-flatcar:app-containers/incus'
```

Comment:
I think base_sysext is not the right place, or? We would want this to be built after the prod image and not be part of it. I would extend `build_image` to call `build_sysext` from a new if-block after `if [[ "${PROD_TAR}" -eq 1 ]]; then` controlled by an argument to build sysexts or not (default could be yes?).
The built sysext images needs to be treated like an OEM sysext, to get uploaded and signed the same way. The naming scheme is `flatcar-NAME.raw`